lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -202,32 +223,26 @@ div.course-wrapper {
         border-left: 4px solid #C93B34 !important;
         margin: 0 auto;
       }
+      &.warning {
+        border-left: 4px solid $warning-color !important;
```

Comment:
`@include border-left(4px solid $warning-color);`

Let's not use `!important` if we don't have to.